src/mozanalysis/experiment.py

```diff
@@ -357,8 +357,8 @@ def _build_enrollments_query(self, time_limits, enrollments_query_type):
         """Return SQL to query a list of enrollments and their branches"""
         if enrollments_query_type == 'normandy':
             return self._build_enrollments_query_normandy(time_limits)
-        elif enrollments_query_type == 'glean':
-            raise NotImplementedError
+        elif enrollments_query_type == 'fenix':
```

Comment:
Hmm; this key tells us to call `_build_enrollments_query_fenix()`, which extracts enrollments out of `org_mozilla_fenix.baseline`, so this code is more specific to Fenix than it is to Mako (and if Nimbus registers the experiment in `ping_info.experiments` then mozanalysis should automagically work for analysing those experiments - though if it sends enrollment events then we can do better than this).

I could switch the key to 'fenix-mako' if we expect the approach in `_build_enrollments_query_fenix()` not to be optimal (or relevant) for Nimbus experiments - I'd also rename the method.

What do you think?

Reply:
Fair point. I do expect Nimbus will send events.

Uh, `fenix-baseline`? :D

Reply:
I chose `fenix-fallback` for the key, and `_build_enrollments_query_fenix_baseline()` for the method name because 'fallback' is what the user needs to know ("use this when you don't have enrollment events") and 'baseline' is the implementation detail that might potentially be swapped out for a different ping in the future